Composer/packages/server/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "author": "",
   "nodemonConfig": {
-    "exec": "cross-env TS_NODE_FILES=true node --max-http-header-size=16000 --inspect=9228 -r ts-node/register src/init.ts",
+    "exec": "yarn build && yarn start",
```

Comment:
why this change?

Reply:
I can chime in here- There are a couple of problems with the dev mode (using `ts-node` instead of `node`). 

1. Workers can only be instantiated with `*.js` files. Node throws an error if the filename arg to the worker is not js. This was the case when running with `ts-node`; the filename would be the .ts file instead of a .js one.
2. Ok, so just make the worker file a js one, problem solved. A new error was thrown about node not being able to resolve `ts-node` when executing the worker. 

TBH, we didn't investigate this really hard and made getting the workers working a priority. I will try to take a look at this to see if we can revert to the original dev script, but I would suggest taking this PR in the meantime.